### DISCUSSION TITLE
放宽评论邮箱检测限制

### DISCUSSION THREAD

**Discussion by Junyan-L** [walinejs/waline#1011] [Q&A] [answered]
如果未登录者填写的邮箱域名是四级域名（比如name@xx.xx.xx.xx）会报错，提示请填写正确的邮件地址。

另外我也出现了类似 #996 以及 #702 中出现的问题，未登录用户评论时只要填写了邮箱就会弹出如下错误
`Class or object doesn't exists. [404 GET https://hgjgqv7h.api.lncldglobal.com/1.1/classes/cache_group_count_user_id_mail]`
这样新一次的评论刷新失败，但leancloud后台能看到新评论，手动刷新也能看到新评论。

**Comment by Mister-Hope** [comment, marked as answer]
四级域名我回去看一下。。。。。😅
